### PR TITLE
Adds functionality to submit comment on ctrl+Enter

### LINKED ISSUE #24302: Submit comments in sidebar by CTRL+Enter
### Steps to reproduce
1. open the files app
2. open the sidebar by clicking a file
3. select the comments tab
4. enter some text
5. hit CTRL+Enter

### Expected behaviour
The form is submitted

### Actual behaviour
Nothing happens, I have to either click the submit button or use tab + enter.

**ownCloud version:**
9.1

@nickvergessen @schiesbn as discussed

### PR DESCRIPTION
**Pull request by vincchan**
closes #24302

### CODE REVIEW

File: apps/comments/js/commentstabview.js [hunk 1]
```diff
@@ -301,6 +301,11 @@
 			var limitExceeded = (len > this._commentMaxLength);
 			$field.toggleClass('error', limitExceeded);
 			$submitButton.prop('disabled', limitExceeded);
+
+			//submits form on ctrlEnter
+			if (ev.keyCode === 13 && ev.ctrlKey) {
+				$submitButton.click();
+		  }
```
Comment (MorrisJobke) [inline apps/comments/js/commentstabview.js]: The indentation seems to be wrong. Please use tabs. Thanks :)

**Comment by MorrisJobke** [comment]
Beside my comment this looks really nice and works 👍

**Comment by MorrisJobke** [comment]
What I would really like is to also trigger this when the command key and enter is pressed on OS X (that is the common key stroke there). Maybe you could find out how to do this :)

**Comment by vincchan** [comment]
@MorrisJobke Please have another look at it, thanks! :)

**Comment by MorrisJobke** [comment]
Really nice! Works! I like it 👍

**Comment by MorrisJobke** [comment]
cc @PVince81 @blizzz for review :)

**Comment by PVince81** [comment]
JS tests still run locally, no need to bother Jenkins.

**Comment by PVince81** [comment]
:+1:

@DeepDiver1975 please merge (see my CI comment above)

**Comment by jancborchardt** [comment]
Very nice, thanks a lot @vincchan! :)

**Comment by vincchan** [comment]
@jancborchardt happy to contribute :)

**Comment by MTRichards** [comment]
Awesome.

**Pull request merged**